### ISSUE TITLE
HitBTC- USD/USDT market fails to submit order

### ISSUE THREAD

**Issue by RC-13** [CoinAlpha/hummingbot#3547]
**Describe the bug**
After setting up the `USD-USDT` market using HitBTC as an exchange and started the strategy, the bot shows that an order was created however it immediately gets an error submitting order due to insufficient funds. However, we have a sufficient fund to place the order:
![image](https://user-images.githubusercontent.com/57189990/121291639-2f4ff000-c91b-11eb-921e-742e0389c904.png)
![image](https://user-images.githubusercontent.com/57189990/121291664-3a0a8500-c91b-11eb-9a36-4fe16ff3feb8.png)
After checking the HitBTC exchange, it appears that the `USD-USDT` market doesn't exist.
![image](https://user-images.githubusercontent.com/57189990/121291406-ccf6ef80-c91a-11eb-9b20-b84a7e6b4903.png)
```
2021-06-09 04:05:57,743 - 3409 - hummingbot.connector.exchange.hitbtc.hitbtc_order_book_tracker - INFO - Initialized order book for USD-USDT. 1/1 completed.
2021-06-09 04:06:00,001 - 3409 - hummingbot.strategy.pure_market_making.pure_market_making - INFO - (USD-USDT) Creating 1 bid orders at (Size, Price): ['15 USD, 0.990665 USDT']
2021-06-09 04:06:00,214 - 3409 - hummingbot.connector.exchange.hitbtc.hitbtc_exchange - NETWORK - Error submitting BUY LIMIT_MAKER order to hitbtc for 15.00 USD-USDT 0.990665 - Insufficient funds.
Traceback (most recent call last):
  File "/home/ubuntu/hitbtc/hummingbot/connector/exchange/hitbtc/hitbtc_exchange.py", line 446, in _create_order
    order_result = await self._api_request("POST", Constants.ENDPOINT["ORDER_CREATE"], api_params, True)
  File "/home/ubuntu/hitbtc/hummingbot/connector/exchange/hitbtc/hitbtc_exchange.py", line 345, in _api_request
    raise HitbtcAPIError(parsed_response)
hummingbot.connector.exchange.hitbtc.hitbtc_utils.HitbtcAPIError: {'error': {'code': 20001, 'message': 'Insufficient funds', 'description': 'Check that the funds are sufficient, given commissions'}}
2021-06-09 04:06:00,215 - 3409 - hummingbot.connector.exchange.hitbtc.hitbtc_exchange - WARNING - Error submitting order to hitbtc - Insufficient funds.
```

**Steps To Reproduce**

Here are the steps to reproduce the issue (see attachments in the section below):
1. Create a basic pure market-making strategy and use HitBTC as the exchange.
2. Choose `USD-USDT` as market and start the strategy.
3. Check the log pane and status window.
...

**Screenshots**
![image](https://user-images.githubusercontent.com/57189990/121291306-a33dc880-c91a-11eb-9478-529e066f9389.png)
![image](https://user-images.githubusercontent.com/57189990/121291319-ab960380-c91a-11eb-9f6f-ab355b4dfd8b.png)

**Release version**
0.39.0

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**

**Comment by TheHolyRoger** [comment]
@RC-13 thank you! Should be fixed here: https://github.com/CoinAlpha/hummingbot/pull/3723

**Comment by TheHolyRoger** [comment]
@dennisocana :)

**Comment by dennisocana** [comment]
Fixed in version 0.42 release. Thanks @TheHolyRoger for contributing!
